src/app-layout/__tests__/runtime-drawers.test.tsx

```diff
@@ -73,6 +74,66 @@ describe('Runtime drawers', () => {
     expect(wrapper.findDrawersTriggers()).toHaveLength(1);
   });
 
+  test('opens registered drawer when defaultActive is set', async () => {
+    const { wrapper } = await renderComponent(<AppLayout />);
+    expect(wrapper.findDrawersTriggers()).toHaveLength(0);
+    expect(wrapper.findActiveDrawer()).toBeFalsy();
+    awsuiPlugins.appLayout.registerDrawer({ ...drawerDefaults, defaultActive: true });
+    await delay();
+    expect(wrapper.findActiveDrawer()!.getElement()).toBeInTheDocument();
+  });
+
+  test('updates active drawer if multiple are registered', async () => {
+    awsuiPlugins.appLayout.registerDrawer({
+      ...drawerDefaults,
+      id: 'first',
+      mountContent: container => (container.textContent = 'first drawer content'),
+    });
+    awsuiPlugins.appLayout.registerDrawer({
+      ...drawerDefaults,
+      id: 'second',
+      defaultActive: true,
+      mountContent: container => (container.textContent = 'second drawer content'),
+    });
+    const { wrapper } = await renderComponent(<AppLayout />);
+    expect(wrapper.findActiveDrawer()!.getElement()).toHaveTextContent('second drawer content');
+  });
+
+  test('keeps currently active drawer if a drawer with lower priority was registered', async () => {
```

Comment:
Could it be the second drawer is open because the second drawer is registered later, not the priority?

Reply:
Good finding, I did two bugs which cancelled each other.

Updated the PR, with the new behavior. Now it is strictly in loading order. Loaded first gets open, all others do not (this is also better for UX to prevent them jumping too much)